invenio_vocabularies/datastreams/transformers.py

```diff
@@ -23,3 +25,33 @@ def apply(self, entry, *args, **kwargs):
                   raises TransformerError in case of errors.
         """
         pass
+
+
+class XmlTransformer(BaseTransformer):
+    @classmethod
+    def _xml_to_etree(cls, xml):
+        """Converts xml to a lxml etree."""
+        return etree.HTML(xml)
+
+    @classmethod
+    def _etree_to_dict(cls, tree):
+        d = {tree.tag: {} if tree.attrib else None}
+        children = list(tree)
+        if children:
+            dd = defaultdict(list)
+            for dc in map(cls._etree_to_dict, children):
+                for k, v in dc.items():
+                    dd[k].append(v)
+            d = {tree.tag: {k: v[0] if len(v) == 1 else v
```

Comment:
if `dd` is empty, do we need:
a) `d = { tree.tag: {} }
b) `d = {}`

I think is correct (a), but just sanity check it.